src/helpers/processOptions.test.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const processOptions = require('./processOptions');
```

Comment:
Idea: can we rename processOptions to AmpConfig? And then instantiate it via:

```
const AmpConfig = require('./AmpConfig')

const config = AmpConfig.load({ /* custom options * / });
```

Wdyt?

Reply:
yes

Reply:
updated